### DISCUSSION TITLE
z80pack Imsai8080est replica .dsk file format

### DISCUSSION THREAD

**Discussion by Gmanxxx** [thehighnibble/thehighnibble.github.io#130] [IMSAI8080esp replica]
Are the Imsai8080esp replica disk files the z80pack "default format", or do I need to specify another one?  Looking at the available list of formats, I don't see one for the Imsai.

**Comment by pfcushing** [comment]
to the best of my knowledge, the Imsai8080 Replica disk files are in the IBM-3740 format which is a 77 track, 26 128 byte sectors per track 8 inch single sided diskette.  

Paul Cushing

**Comment by Gmanxxx** [comment]
Thanks.  I tried using that format with the CPMTools from the z80Pack and was able to read the directory listings of the disk images included with the Imsai clone.  And the CPMTools seem to be creating disk images compatible with the Imsai clone.

But...... When I copy the disk images to my PC, they have a file length of 512K bytes.  But the Ibm-3740 format is a lot less, 241K.  Any reason for the mismatch?

**Reply by pfcushing** [reply]
I have CPMTools on a Windows machine and all of the disk images are 256,256 bytes

Paul Cushing

**Comment by Gmanxxx** [comment]
Agreed.  I must have been dreaming or drunk when I thought it was 512K bytes.
